src/content/plots/PlotDownload.service.tsx

```diff
@@ -0,0 +1,56 @@
+import downloadjs from 'downloadjs'
+import html2canvas from 'html2canvas'
+import * as d3 from 'd3'
+
+const PlotDownloadService = {
+    downloadAsPNG: async function () {
+        const cornerPlotElmt = document.querySelector<HTMLElement>('.corner-plot')
+        if (!cornerPlotElmt) return
+        changeColours('white', 'black')
+
+        const canvas = await html2canvas(cornerPlotElmt)
+        const dataURL = canvas.toDataURL('image/png')
+        downloadjs(dataURL, 'corner-plot.png', 'image/png')
+
+        changeColours('#070C27', 'white')
+    },
+    downloadAsSVG: function () {
```

Comment:
```suggestion
    downloadAsSVG: () => {
```
Same as line 6, please use arrow functions for consistency